packages/react-native-reanimated/apple/reanimated/apple/native/PlatformDepMethodsHolderImpl.mm

```diff
@@ -85,17 +85,15 @@ SetGestureStateFunction makeSetGestureStateFunctionBridgeless(RCTModuleRegistry
 
 RequestRenderFunction makeRequestRender(REANodesManager *nodesManager)
 {
-  auto requestRender = [nodesManager](std::function<void(double)> onRender, jsi::Runtime &rt) {
+  auto requestRender = [nodesManager](std::function<void(double)> onRender) {
     [nodesManager postOnAnimation:^(READisplayLink *displayLink) {
 #if !TARGET_OS_OSX
       auto targetTimestamp = displayLink.targetTimestamp;
 #else
       // TODO macOS targetTimestamp isn't available on macOS
       auto targetTimestamp = displayLink.timestamp + displayLink.duration;
 #endif
-      double frameTimestamp =
-
-          (targetTimestamp) * 1000;
+      const double frameTimestamp = targetTimestamp * 1000;
```

Comment:
We purposefully left `(targetTimestamp) * 1000` here in a previous PR because we suspect there used to be a function call here that accepted `targetTimestamp` as the first argument. Most likely this was related to slow animations. I'm okay with this change, just wanted to remind about it. Perhaps we should fix it in a separate PR?

Reply:
I've restored those changes and I'll prepare a PR with the fix 👍